apps/aechannel/src/aesc_state_cache.erl

```diff
@@ -197,7 +200,7 @@ transform_record_to_encrypted_form_with_default_password(#pch{id = ChId, pubkeys
     [encrypt_cache(Cache1)];
 transform_record_to_encrypted_form_with_default_password(#pch{pubkeys = [Pub1, Pub2]} = Record) ->
     Records = [Record#pch{pubkeys = [Pub1]}, Record#pch{pubkeys = [Pub2]}],
-    lists:flatten([transform_record_to_encrypted_form_with_default_password(Record) || Record <- Records]).
+    lists:flatten([transform_record_to_encrypted_form_with_default_password(R) || R <- Records]).
```

Comment:
The migrated records will contain an undefined opts map. We need to create a opts map here.

Reply:
The default value for `opts` in `#ch_cache{}` is `#{}`...